lua/core/telescope.lua

```diff
@@ -96,7 +96,7 @@ function M.find_lunarvim_files(opts)
     },
     prompt = ">> ",
     prompt_title = "~ LunarVim files ~",
-    cwd = CONFIG_PATH,
+    cwd = require("utils").join_paths(LUNARVIM_RUNTIME_DIR, "lvim"),
```

Comment:
Can we store `utils` in a local variable ? I don't mind when we call a single function, but here it's used multiple times.
https://github.com/luarocks/lua-style-guide#modules